pkgs/os-specific/windows/mingw-w64/osvi.patch

```diff
@@ -0,0 +1,11 @@
+--- a/mingw-w64-headers/include/multimon.h
++++ b/mingw-w64-headers/include/multimon.h
+@@ -127,7 +127,7 @@
+   WINBOOL IsPlatformNT() {
+     OSVERSIONINFOA oi = { 0 };
+ 
+-    oi.dwOSVersionInfoSize = sizeof (osvi);
```

Comment:
Why do we do this? A comment in the source would be nice. Lest the next one reading this will be puzzling over this...

Reply:
It's a bug I think. See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863064.